src/test/resources/proxy/requests-search.json

```diff
@@ -226,5 +226,92 @@
         ]
       }
     }
+  },
+  {
+    "name": "GET search from root with queryparameter",
+    "requestGateway": {
+      "method": "GET",
+      "url": "_search?q=age:>33",
+      "body": null
+    },
+    "requestOpensearch": {
+      "method": "GET",
+      "url": "_search?q=age:>33",
```

Comment:
workspace-mockname-index-3/_search

Reply:
to follow the pattern and intention, it would be workspace-mockname-*/_search

Which revealed a bug, in the case of _search, then and only then should URL prefixer return `workspace-mockname-*/_search`